### ISSUE TITLE
vendor:publish --provider not found

### ISSUE THREAD

**Issue by necrogami** [laravel-zero/laravel-zero#421]
```
php application vendor:publish --provider "LaravelZero\Framework\Components\Updater\Provider"
[2022-07-19 02:44:00] development.ERROR: The "--provider" option does not exist. {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): The \"--provider\" option does not exist. at /home/necrogami/code/server-client/vendor/symfony/console/Input/ArgvInput.php:220)
[stacktrace]
#0 /home/necrogami/code/server-client/vendor/symfony/console/Input/ArgvInput.php(149): Symfony\\Component\\Console\\Input\\ArgvInput->addLongOption()
#1 /home/necrogami/code/server-client/vendor/symfony/console/Input/ArgvInput.php(82): Symfony\\Component\\Console\\Input\\ArgvInput->parseLongOption()
#2 /home/necrogami/code/server-client/vendor/symfony/console/Input/ArgvInput.php(71): Symfony\\Component\\Console\\Input\\ArgvInput->parseToken()
#3 /home/necrogami/code/server-client/vendor/symfony/console/Input/Input.php(55): Symfony\\Component\\Console\\Input\\ArgvInput->parse()
#4 /home/necrogami/code/server-client/vendor/symfony/console/Command/Command.php(267): Symfony\\Component\\Console\\Input\\Input->bind()
#5 /home/necrogami/code/server-client/vendor/symfony/console/Application.php(998): Symfony\\Component\\Console\\Command\\Command->run()
#6 /home/necrogami/code/server-client/vendor/symfony/console/Application.php(299): Symfony\\Component\\Console\\Application->doRunCommand()
#7 /home/necrogami/code/server-client/vendor/symfony/console/Application.php(171): Symfony\\Component\\Console\\Application->doRun()
#8 /home/necrogami/code/server-client/vendor/illuminate/console/Application.php(102): Symfony\\Component\\Console\\Application->run()
#9 /home/necrogami/code/server-client/vendor/laravel-zero/foundation/src/Illuminate/Foundation/Console/Kernel.php(129): Illuminate\\Console\\Application->run()
#10 /home/necrogami/code/server-client/vendor/laravel-zero/framework/src/Kernel.php(96): Illuminate\\Foundation\\Console\\Kernel->handle()
#11 /home/necrogami/code/server-client/dns-taxi(37): LaravelZero\\Framework\\Kernel->handle()
#12 {main}
"}


  The "--provider" option does not exist.
```

When following the manual for updating the updater strategy it tells us to vendor publish the updater so we can update the config. However trying to execute the command comes up that it doesn't exist.

**Comment by theVosCache** [comment]
+1

It looks like the whole "vendor:publish" command is not loaded

**Comment by owenvoke** [comment]
Hi, it looks like this was removed accidentally (or intentionally, @nunomaduro?) in this commit [`3b351ca`](https://github.com/laravel-zero/laravel-zero/commit/3b351caf6a78983b92dbd8be939a3b89db55510d#diff-c777e2a3a26868d9f361dcadefdb881c193eac8510b243f39d008b0bfb9878e6L62). I've re-added this in https://github.com/laravel-zero/laravel-zero/commit/f5454bda728c69f078590422b49e244bc3447119. 👍🏻

However, the issues with hiding console commands need to be addressed. I'll take a look at https://github.com/laravel-zero/framework/pull/465 and see if we can get that merged as well. 👍🏻

**Comment by owenvoke** [comment]
Closed as part of v9.1.3 👍🏻
